packages/grafana-ui/src/components/Toggletip/Toggletip.test.tsx

```diff
@@ -140,43 +140,6 @@ describe('Toggletip', () => {
     expect(onOpen).toHaveBeenCalledTimes(1);
   });
 
-  it('should be able to focus toggletip content next in DOM order - forwards and backwards', async () => {
```

Comment:
maybe should we have essentially the updated version of this test, ensuring that focus is trapped within it?

Reply:
yeah good shout, i can do that 👍